specification/schemas/Zone.json

```diff
@@ -0,0 +1,67 @@
+{
+  "allOf": [
+    {
+      "$ref": "#/components/schemas/ZoneResource"
+    },
+    {
+      "properties": {
+        "attributes": {
+          "type": "object",
+          "required": [
+            "postal_code",
+            "country_code",
+            "code",
+            "name"
+          ],
+          "additionalProperties": false,
+          "properties": {
+            "postal_code": {
+              "type": "array",
+              "items": {
+                "type": "string",
+                "example": "2131 BC"
+              }
+            },
```

Comment:
This should be a string + let's give it a description mentioning it's a regex. We can actually re-use the existing schema from ServiceRegion. Let's put below in a new file and use it as `"$ref": "#/components/schemas/PostalCodeRegex"`

```
    "postal_code": {
      "type": "string",
      "example": "^((GY|JE).*|TR2[1-5]) ?[0-9]{1}[A-Z]{2}$",
      "description": "Regular expression matching postal code areas."
    }
```